lib/global_setup.ts

```diff
@@ -0,0 +1,58 @@
+import { chromium } from '@playwright/test';
+import { config as testConfig } from './config';
+import { ConsoleDotAuthPage } from '@lib/pom/auth';
+import config from '../playwright.config';
+
+export default async function globalSetup() {
+  const users = [
+    {
+      name: testConfig.adminUsername,
+      password: testConfig.adminPassword,
+      path: testConfig.adminAuthFile
+    },
+    {
+      name: testConfig.username,
+      password: testConfig.password,
+      path: testConfig.user1AuthFile
+    },
+    {
+      name: testConfig.username_2,
+      password: testConfig.password_2,
+      path: testConfig.user2AuthFile
+    },
+    {
+      name: testConfig.stratosphere1username,
+      password: testConfig.stratospherePassword,
+      path: testConfig.stratosphere1AuthFile
+    },
+    {
+      name: testConfig.stratosphere2username,
+      password: testConfig.stratospherePassword,
+      path: testConfig.stratosphere2AuthFile
+    },
+    {
+      name: testConfig.stratosphere3username,
+      password: testConfig.stratospherePassword,
+      path: testConfig.stratosphere3AuthFile
+    },
+    {
+      name: testConfig.stratosphere4username,
+      password: testConfig.stratospherePassword,
+      path: testConfig.stratosphere4AuthFile
+    }
+  ];
+
+  for (const user of users) {
+    if (user.name != undefined) {
+      const browser = await chromium.launch();
```

Comment:
what about Firefox or other browsers ?

Reply:
Sessions are opened in firefox if specified so this is just for login. I don't think we actually care about this as it is functionality of console dot and  not RHOSAK